opensm/osm_console.c

```diff
@@ -1658,7 +1658,7 @@ static void help_version(FILE * out, int detail)
 
 static void version_parse(char **p_last, osm_opensm_t * p_osm, FILE * out)
 {
-	fprintf(out, "%s build %s %s\n", p_osm->osm_version, __DATE__, __TIME__);
+	fprintf(out, "%s build\n", p_osm->osm_version);
```

Comment:
Probably " build" should go away, and "%s\n" should remain.

Reply:
duh, thanks for nothing dyslexia

Reply:
French vs English - it's quite normal. Not a diagnosis :-)